c/src/space.rs

```diff
@@ -39,7 +39,7 @@ impl space_t {
         unsafe{ Box::from_raw(self.space.cast_mut()).0 }
     }
     pub(crate) fn borrow(&self) -> &DynSpace {
-        unsafe{ &(*self.space).0 }
+        unsafe{ &(&*self.space).0 }
```

Comment:
FYI, this change should make no difference whatsoever to the behavior, but it more clearly communicates how I like to think about what is happening.